03-functions-and-loops/invest.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def invest(amount, rate, num_of_years):
+    initial_amount = amount
+    print("\nPrincipal amount: ", amount)
+
+    print("annual return rate: ", str(rate))
+
+    for year in range(1, num_of_years + 1):
+        compound_rate = amount * rate
+        amount = amount + compound_rate
+        print("year ", num_of_years, ":", amount)
+
+    roi  = amount - initial_amount
+    print("you made in {} years ${}".format(num_of_years, roi))
+
+def main():
+    invest(100, .05, 8)
```

Comment:
When a function has several parameters, use keyword arguments (as stated in the code style guide).